shap/benchmark/metrics.py

```diff
@@ -42,22 +42,23 @@ def runtime(X, y, model_generator, method_name):
         model.fit(X_train, y_train)
 
         # evaluate each method
-        start = time.time()
-        explainer = getattr(methods, method_name)(model, X_train)
-        build_time = time.time() - start
+        # start = time.time()
```

Comment:
remove commented code

Reply:
These were originally commented out to test the old API, but the old API can remain for now so will be uncommented